project.mml

```diff
@@ -2196,52 +2188,80 @@ Layer:
     <<: *extents
     Datasource:
       <<: *osm2pgsql
-      table: |-
+      table: &amenity_low_priority_sql |-
         (SELECT
             way,
+            name,
             COALESCE(
-              'highway_' || CASE WHEN highway IN ('mini_roundabout') THEN highway ELSE NULL END,
```

Comment:
You are newly allowing mapping various things with polygons which were previously only rendered on nodes.  This should not be part of the bugfix PR.

Reply:
Which ones? I thought I caught them all with `AND way_area IS NULL` - only barriers and parking (not parking entrance) should be allowed if way_area is not null?

Reply:
I already made test images which show that wayside_shrine and crosses do not render when mapped as areas, so I believe `AND way_area IS NULL` is working, as in `amenity-points`

Reply:
Oh, sorry - my bad.